T_base_cam:
  matrix:
  - - -0.9996818408996785
    - -0.005912628684227844
    - -0.02452055866966396
    - 0.18923424770394076
  - - -0.015896164979188332
    - -0.607103905190684
    - 0.7944634417272927
    - -0.1946873394772125
  - - -0.019583894259937396
    - 0.7946004587994292
    - 0.6068167614373816
    - -0.08543459010335587
  - - 0.0
    - 0.0
    - 0.0
    - 1.0
T_cam_base:
  matrix:
  - - -0.9996818408996785
    - -0.015896164979188332
    - -0.019583894259937396
    - 0.18440611705952703
  - - -0.005912628684227844
    - -0.607103905190684
    - 0.7946004587994292
    - -0.049190207753319795
  - - -0.02452055866966396
    - 0.7944634417272927
    - 0.6068167614373816
    - 0.21115524453617884
  - - 0.0
    - 0.0
    - 0.0
    - 1.0
frames:
  child: camera
  parent: base
rotation_base_cam:
  w: -0.0027845711976539262
  x: -0.012301451678804892
  y: 0.44321585437335526
  z: 0.8963261833071351
rotation_cam_base:
  w: 0.0027845711976539262
  x: -0.012301451678804892
  y: 0.44321585437335526
  z: 0.8963261833071351
stamp: '2025-08-12T14:08:07.142944+00:00'
translation_base_cam:
  x: 0.18923424770394076
  y: -0.1946873394772125
  z: -0.08543459010335587
translation_cam_base:
  x: 0.18440611705952703
  y: -0.049190207753319795
  z: 0.21115524453617884
